### PR TITLE
Blog post on open comminication

### PR DESCRIPTION
**Pull request by tnt-wg2**
Halla. can you take a look on the blog post and comment thanks.

### CODE REVIEW

File: pages/blog/_posts/open communication.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# What are your weekend plans?
```
Comment (david-wg2) [inline pages/blog/_posts/open communication.md]: This section is missing a paragraph. It looks like the paragraph belonging to `Open communication as a tool to break down barriers` would fit well here.

File: pages/blog/_posts/open communication.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## We are all just humans  
+Every one of my coworkers are humans in different stages of life. Some are fresh out of college, some are getting married, some are establishing a family, and others getting a dog. But how much do you actually know about your coworkers, and do you take the time to get to know them? 
+
+## Going to the doctor
+We openly share non-work related information on the same level as work-related information. Everything from doctor appointments, that we will run late for office, that we deliver or pick up kids in kindergarten, veterinarian appointments, and we add our sports games and events in our calendar. 
+
+You might think this is information-overload in an already hectic day. Nonetheless, it provides opportunities. Opportunities to see what is happening in my co-worker's life and we connect on a personal level. It provides possibilities to follow up about their sports games, and asks how their kids are doing. 
+
+## I do not want to be perceived as a “Mom”
```
Comment (david-wg2) [inline pages/blog/_posts/open communication.md]: This is a quote, so maybe it should be `"I do not want to be perceived as a 'Mom'"`

File: pages/blog/_posts/open communication.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Being in a company where my male colleges openly share that need to leave early to pick up kids or sit with their kids in video meetings breaks down role barriers. It makes it OK to share the information, OK to leave the office, run from meetings, and have a bad day. 
+
+Getting to know your colleagues on a personal level builds interpersonal trust and emotional connections between team members. It fosters the foundation for a culture that embraces vulnerability and the fact that it is allowed to f… up. If f… ups happen, then you know that your coworker will support you. 
+
+Interpersonal trust and emotional connections, built on open and informal communication, are proven positive for creativity and innovation in a company. 
+
+So, do you know what your coworkers are doing this weekend? 
+
+
```
Comment (david-wg2) [inline pages/blog/_posts/open communication.md]: Remove empty line.

File: pages/blog/_posts/open communication.md (hunk 1 above)
Comment (tnt-wg2) [reply pages/blog/_posts/open communication.md]: it is the headline. Might only need it in the File name???

**Pull request merged**